bats/fb-content-katello.bats

```diff
@@ -121,8 +121,13 @@ EOF
     rpm -e `rpm -qf /etc/rhsm/ca/candlepin-local.pem`
   fi
 
+  yum erase -y 'katello-ca-consumer-*' || true
   rpm -Uvh http://localhost/pub/katello-ca-consumer-latest.noarch.rpm || true
   subscription-manager register --force --org="Test_Organization" --activationkey="Test AK" || true
+  if tIsRHEL && ! tIsCentOSCompatible; then
+    # skipping the rest of the test as we don't provide RHEL subs
+    skip
+  fi
```

Comment:
It fails on RHEL as we don't provide a subscription for RHEL itself, so the output will be "Unknown".
As an alternative we could look at `sub-man list --consumed` and verify that lists the subs we actually added to the AK earlier.